systems/registry/invitation/pkg/server/invitation.go

```diff
@@ -61,11 +61,11 @@ func NewInvitationServer(iRepo db.InvitationRepo, invitationExpiryTime uint, aut
 	}
 }
 
-func (i *InvitationServer) Add(ctx context.Context, req *pb.AddInvitationRequest) (*pb.AddInvitationResponse, error) {
+func (i *InvitationServer) Add(ctx context.Context, req *pb.AddRequest) (*pb.AddResponse, error) {
 	log.Infof("Adding invitation %v", req)
 	invitationId := uuid.NewV4()
 
-	if req.GetOrg() == "" || req.GetEmail() == "" || req.GetName() == "" {
+	if i.orgName == "" || req.GetEmail() == "" || i.orgName == "" {
```

Comment:
Why are we checking `i.orgName `  twice?  Shouldn't be the third check be `req.GetName()` instead?